unison-src/transcripts/upgrade-sad-path.output.md

```diff
@@ -35,6 +35,23 @@ proj/main> upgrade old new
   I couldn't automatically upgrade old to new. However, I've
   added the definitions that need attention to the top of
   scratch.u.
+  
+  When you're done, you can run
+  
+    * switch /main
+    * merge /upgrade-old-to-new
+    * delete.branch /upgrade-old-to-new
+  
+  or (equivalently)
+  
+    * upgrade.commit
+  
+  to merge your changes back into main. Or, if you'd like to
+  abandon the upgrade instead, you can run
+  
+    * delete.branch /upgrade-old-to-new
+  
+  to delete upgrade-old-to-new and switch back to main.
```

Comment:
I'm looking with fresh eyes today and thinking we should just promote the new command now instead of describing the commands you won't want to run:

```suggestion
  When you're done, you can run
  
    upgrade.commit
  
  to merge your changes back into main and delete the temporary branch.
  Or, if you need to abandon the upgrade instead, you can run
  
    delete.branch /upgrade-old-to-new
  
  to delete the temporary branch and switch back to main.
```

Reply:
I thought Github would highlight the actual changes, but I guess they only do that for single-line suggestions.